src/main/resources/i18n/messages.properties

```
@@ -682,3 +682,4 @@ Wizard.9=Wired (100 Megabit)
 Wizard.10=Wireless
 Wizard.11=Should UMS hide its advanced options? They can be enabled later
 Wizard.12=Finally, choose a folder to share. You can choose more later on the Navigation/Share Settings tab.\n\nIf you don't choose a folder, we'll choose a default based on your operating system.
+Wizard.13=Would you like to configure UMS with the default configuration?
```

I think it would be good to reverse the logic of the question so it is like:

```suggestion
Wizard.13=<html><strong>Would you like to run the first-time configuration wizard?</strong><br>This is a quick way to get the best experience from UMS.</html>
```

and the the buttons could use custom labels:
`Yes, run the wizard" and "No, use the defaults"

What do you think?
Do any of the @UniversalMediaServer/developers have input about that?

---

I like the presentation you have here, should I implement the change or wait for other input?